cmd/provisioner-localpv/app/provisioner_blockdevice.go

```diff
@@ -93,7 +102,7 @@ func (p *Provisioner) ProvisionBlockDevice(ctx context.Context, opts pvControlle
 		WithAccessModes(pvc.Spec.AccessModes).
 		WithCapacityQty(pvc.Spec.Resources.Requests[v1.ResourceName(v1.ResourceStorage)]).
 		WithLocalHostPathFormat(path, fsType).
-		WithNodeAffinity(nodeHostname)
+		WithNodeAffinity(nodeAffinityKey, nodeAffinityValue)
```

Comment:
Does this support 2 params? Shouldn't we pass a map here as well?

Reply:
Yes, the v1 version of the package `"github.com/openebs/dynamic-localpv-provisioner/pkg/kubernetes/api/core/v1/persistentvolume"` supports this.

Reply:
Map for what purpose?

Reply:
Single parameter

Reply:
We can change the function as necessary.. The function currently is written that way. What do you suggest?

Reply:
Let it be then. Thanks.